server/models/Request.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+
+const RequestSchema = new mongoose.Schema({
+    userId:{
+        type: String,
+        required: true,
+      },
+    sitterId:{
+        type: String,
+        required: true,
+    },
```

Comment:
[This comment](https://github.com/hatchways/team-french-onion/pull/47#discussion_r718392115)